frontend/app/controllers/spree/orders_controller.rb

```diff
@@ -5,6 +5,7 @@ class OrdersController < Spree::StoreController
     helper 'spree/products', 'spree/orders'
 
     respond_to :html
+    respond_to :js, only: [:populate, :update]
```

Comment:
I can actually get rid of this one if you prefer, as one can easily just add the line in a decorator.